types.go

```diff
@@ -27,6 +27,7 @@ type Arch string
 // Additional architectures permitted to be used for system calls
 // By default only the native architecture of the kernel is permitted
 const (
+	ArchNative      Arch = "SCMP_ARCH_NATIVE"
```

Comment:
Isn't arch native always referring to the host arch? @giuseppe ?

Reply:
Yes.

From my understanding, it's for removing the host architecture from the Seccomp filter (in case you want a Seccomp filter that only includes syscalls for one architecture, maybe?)

Reply:
`SCMP_ARCH_NATIVE` is always the native arch/ABI and exists to make it easier to call the libseccomp APIs that require an ARCH token (the code doesn't need to know the host's arch/ABI).

Reply:
Does this mean we should add it?

Reply:
I interpreted it like this, or are there any reasons to explicitly not support it?

Reply:
I'm not sure what harm it would cause.  It's part of the core libseccomp API, it isn't going anywhere so it is safe to use.